### DISCUSSION TITLE
对拖动条颜色值的疑问

### DISCUSSION THREAD

**Discussion by techfisha** [FinnRaze/mpv-osc-modern-f#7] [General]
作者你好，我看到最近的一次提交加入了进度条颜色的选项，我试了下 osc-modern 原脚本里的蓝色代码 “HE39C42” ，但这个颜色代码在网页上查出来是橙色，但是写到配置里却是蓝色，后来我又去看了其他网页播放器的进度条，比如网页播放器里的蓝色"00aeec" ,在mpv里就显示成了棕/黄色，请问这是什么原因呢? 

另外，我发现 conf 配置文件里这样写 seekbarfg_color="HE39C42" 会导致整个进度条变黑，要写成  seekbarfg_color = "HE39C42" 才会正常

**Comment by techz1** [comment]
MPV上游项目采用ass标准顺序：BGR

“Note: colors are given as hexadecimal values and use ASS tag order: BBGGRR (blue green red)”
